core-library/src/main/java/org/silverpeas/core/admin/domain/synchro/SynchroGroupReport.java

```diff
@@ -118,6 +119,14 @@ static public void debug(String classe, String message) {
     }
   }
 
+  static public void debug(String classe, Supplier<String> msgSupplier) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)